packages/gasket-plugin-intl/package.json

```diff
@@ -41,6 +41,7 @@
     "@gasket/helper-intl": "^6.38.1",
     "@hapi/accept": "^6.0.0",
     "fs-extra": "^10.0.0",
+    "glob": "^7.2.0",
```

Comment:
Ugh, good find. I'm surprised we hadn't encountered this before.

However, this should be at `^8.0.0` to align with versions in the other plugins in the monorepo.

Reply:
Checking if it's compatible with `promisify`...

Reply:
Yep, that version also works. Updated.